common/lib/common-utils/package.json

```diff
@@ -34,7 +34,7 @@
 		"ci:build:docs": "api-extractor run --typescript-compiler-folder ./node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
 		"ci:test": "npm run test:report",
 		"ci:test:coverage": "npm run test:coverage",
-		"clean": "rimraf --glob _api-extractor-temp dist lib *.tsbuildinfo *.build.log",
+		"clean": "rimraf --glob _api-extractor-temp dist lib *.tsbuildinfo *.build.log nyc",
```

Comment:
Should we clean this up to use quoted patterns too? I think it won't work on Windows otherwise.